test/index.js

```diff
@@ -113,6 +113,64 @@ const exportTest = (methodName, genArgs, testFn, extraName = '') =>
 const exportBenchmark = (methodName, genArgs, useTryCatch = false, extraName = '') =>
   exports[`${methodName}${extraName}Benchmark`] = benchmarkGen(methodName, genArgs, useTryCatch, extraName)
 
+// h3IsValid has unique input parsing logic to return false rather than throw on invalid input
+exports['h3IsValid_array'] = test => {
+    test.ok(h3node.h3IsValid([0x3fffffff, 0x8528347]), 'Integer H3 index is considered an index');
```

Comment:
Nit: For methods expected to return boolean, prefer `t.equal(result, false)` etc to avoid tests passing on truthy/falsy values like `undefined` etc.